js/main.js

```diff
@@ -1,10 +1,11 @@
 $("#more").click(function(){
     $("section").css({"position": "fixed"});
-    $("#info").animate({"left": "10%",}, 200);
-    $("#info").animate({"left": "0%",}, 200);
+    $("#info").animate({"left": "0%"}, 200);
```

Comment:
Not talking about the bounce effect. I'm talking about the shift left and the opacity change. They don't happen simultaneously and end up looking stilted.